### PR TITLE
Add validator in the recipients step in draft wizard.

### PR DESCRIPTION
**Pull request by radiansz**

### CODE REVIEW

File: src/blocks/form/draftChooseRecipients/draftChooseRecipients.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 define(function(require) {
 
     var Form = require('blocks/form/form');
-        ContentTemplate = require('tpl!./draftChooseRecipients.ejs')
+        ContentTemplate = require('tpl!./draftChooseRecipients.ejs');
```
Comment (zouxuoz) [inline src/blocks/form/draftChooseRecipients/draftChooseRecipients.js]: missed `var`

File: src/blocks/form/draftChooseRecipients/draftChooseRecipients.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+    var RequireValidator = require('kit/validators/required');
```
Comment (zouxuoz) [inline src/blocks/form/draftChooseRecipients/draftChooseRecipients.js]: may be `RequiredValidator`?

**Comment by trikashny** [comment]
LGTM

**Comment by rosendi** [comment]
@radiansz I completely agree with the @zouxuoz comments. Once they are fixed it will be merged.

P.S. Rename the commit's comment into`fix(mailings): add required validator for the recipients`

**Pull request merged**